<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomizeDialog</class>
 <widget class="QDialog" name="CustomizeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="CustomizeLabel">
         <property name="minimumSize">
          <size>
           <width>153</width>
           <height>18</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Select Your Color:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="TankLabel">
         <property name="minimumSize">
          <size>
           <width>311</width>
           <height>291</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="RedButton">
         <property name="text">
          <string>Red</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PinkButton">
         <property name="text">
          <string>Pink</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CyanButton">
         <property name="text">
          <string>Cyan</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="YellowButton">
         <property name="text">
          <string>Yellow</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="GreenButton">
         <property name="text">
          <string>Green</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BlueButton">
         <property name="text">
          <string>Blue</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PurpleButton">
         <property name="text">
          <string>Purple</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BlackButton">
         <property name="text">
          <string>Black</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="custSaveButton">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="custCloseButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
